Cesium.CodeGen.Tests/CodeGenTestBase.cs

```diff
@@ -10,6 +10,7 @@
 
 namespace Cesium.CodeGen.Tests;
 
+[UseCulture("")]
```

Comment:
I'm curious why explicit control of the culture is needed here? Isn't we should write code which runs on all cultures in same way.

Reply:
[Instruction.ToString()](https://github.com/jbevain/cecil/blob/49b1c52df5e8d2bf03768dc35ba6575e88277bc7/Mono.Cecil.Cil/Instruction.cs#L95) uses the culture of the thread, which can lead to different results on different machines and fail tests

Reply:
Cool that you submit https://github.com/jbevain/cecil/pull/870 to fix underlying root cause. Would be interesting to know how it would be responded.